proposals/boost/boost-maven/boost-boms/booster-ee7-bom/pom.xml

```diff
@@ -43,7 +43,7 @@
 				<!-- Booster version distinct from the BOM, boost-maven-plugin version -->
 				<!-- Could also use 7.0, 8.0 for EE7, 8 -->
 				<!-- Probably will be 1.0 for initial GA -->
-				<version>0.1-SNAPSHOT</version>
+				<version>0.2.0-SNAPSHOT</version>
 				<scope>provided</scope>
```

Comment:
Comment above is now outdated... I think a better way to pattern versioning after the technology version might be to do a 2.0-alpha1, 2.0-beta1, etc.... which both indicates that this is pre-GA but also allows for real releases.